test/jdk/javax/swing/plaf/basic/BasicSliderUI/bug4419255.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2001, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.Color;
+import javax.swing.JColorChooser;
+import javax.swing.UIManager;
+
+import jtreg.SkippedException;
+
+/*
+ * @test
+ * @bug 4419255
+ * @library /java/awt/regtesthelpers /test/lib
+ * @build PassFailJFrame
+ * @summary Tests if Metal Slider's thumb isn't clipped
+ * @run main/manual bug4419255
+ */
+
+public class bug4419255 {
+
+    public static void main(String[] args) throws Exception {
+
+        // ColorChooser UI design is different for GTK L&F.
+        // There is no RGB tab available for GTK L&F, skip the testing.
+        if (UIManager.getLookAndFeel().getName().contains("GTK")) {
+            System.out.println("Test skipped for " + UIManager.getLookAndFeel().getName());
```

Comment:
guess no need to enquire it again in as we know it's GTK

Reply:
I think print statement is not required, anyways it is captured in exception, so removed.